pages/supershardians/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, Button, Container, filter, HStack, SimpleGrid } from "@chakra-ui/react";
+import Card from "components/common/ProfileInfoCard";
+import Footer from "components/sections/Footer";
+import Hero from "components/sections/Hero";
+import BeASuperShardian from "components/sections/supershardians/BeASuperShardians";
+import Shardians from "components/sections/supershardians/ShardiansList";
+import JoinCommunity from "components/sections/JoinCommunity";
+import { InferGetStaticPropsType, NextApiResponse, NextPage } from "next";
+import Image from "next/image";
+import React, { useEffect, useState } from "react";
+import { getSuperShardians } from "utils/api";
+
+const buttonCategories = [
+  { name: "All", selected: true },
+  { name: "Content", selected: false },
+  { name: "Design", selected: false },
+  { name: "Marketing", selected: false },
+  { name: "Technology", selected: false },
+];
+
+const SuperShardians = ({
+  superShardians,
+}: InferGetStaticPropsType<typeof getStaticProps>): React.ReactNode => {
+  const [categories, setCategories] = useState(buttonCategories);
+  const [filteredData, setFilteredData] = useState([] as any);
+
+  useEffect(() => {
+    setFilteredData(superShardians);
+  }, []);
+
+  const onCategoryChangeHandler = (event: any) => {
+    if (event) {
+      setFilteredData(
+        superShardians.filter(
+          (data: any) => event.target.value === "All" || data.category[0] === event.target.value
+        )
+      );
+      setCategories(
+        categories.map((category) =>
+          event.target?.value === category.name
+            ? { selected: true, name: category.name }
+            : { selected: false, name: category.name }
+        )
+      );
+    }
+  };
+  return (
+    <>
```

Comment:
We should also have SEO tags <meta > for this page like in other pages. Pls contact PM's and add the tags here..

Reply:
PMs would take more time to provide these tag, Point noted I will have these changes along with translation PR.

Reply:
This is done